ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -85,7 +85,6 @@ export interface TimeChartProps {
   tooltipConfig?: TooltipConfig;
   noDataVariant?: 'chart' | 'message';
   syncGroup?: string;
-  isStackedBar?: boolean;
```

Comment:
Does the removal of this prop justify making this PR a breaking change or is this such a new feature that it's fine not to worry about it?